packages/unimodules-test-core/babel.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = function(api) {
+  api.cache(true);
+  return {
+    presets: ['babel-preset-expo'],
+  };
+};
```

Comment:
I believe we don't want to have any JS level files in this module 🤔 at least for now?

Reply:
Yeah, you're right. I got rid of those.